lib/Ecomail/ChimpDrill/ChimpDrill.php

```diff
@@ -85,8 +85,7 @@ protected function parseMessage()
                 $message = preg_replace_callback($pattern, array($this, $method), $message);
             }
 
-            // Write file
-            $file = tempnam(storage_path('temp'), 'chimpdrill-');
+            $file = @tempnam(storage_path('temp'), 'chimpdrill-');
```

Comment:
The `@` silences errors. We're getting this error: https://github.com/Glavin001/atom-beautify/issues/1108

Although I can't find any documented change to `tempnam`, I guess this used to be a warning. When I tried testing this on our local platform, `storage_path` returned `/var/www/storage/temp`, which seems okay?

Reply:
Since this path won't be writeable in prod, can we do `sys_get_temp_dir()` instead of `storage_path('temp')`?